pyxem/signals/vdf_image.py

```diff
@@ -30,3 +38,109 @@ def __init__(self, *args, **kwargs):
         self, args, kwargs = push_metadata_through(self, *args, **kwargs)
         super().__init__(*args, **kwargs)
         self.vectors = None
+
+    def get_vdf_segments(self, min_distance=1, min_size=1,
```

Comment:
I think this should return another VDFImage object - it's basically the same thing diffracted intensity as a function of position with associated vectors. Is that ok @tinabe ?

Reply:
Yes, that is all right

Reply:
ok, for now I think we will leave this as it is.